.github/workflows/test.yml

```diff
@@ -51,6 +50,67 @@ jobs:
         # need to figure out what happened on mac with the utilities
         run: python -m pymmcore_plus.install -r 20220420
 
+      - name: Install Linux Micro-Manager build tools and dependencies
```

Comment:
let's split all this out into a separate `test_linux` test, then we can do away with the `if: runner.os` in most of the steps... Also we don't need to parametrize quite so much (particularly given how much slower it is than the other tests), so let's just test `ubuntu-latest` with python 3.10 on pyqt5 (or, if you'd like to parametrize either the backend or the python version x2 that's fine)